src/main/java/frc/robot/subsystems/CargoHolder.java

```diff
@@ -34,8 +35,11 @@ public void setIsCargoCollected(boolean isCollected){
   }
 
   public boolean isCargoCollectedStall() {
-    //TODO finish this.
-    return true;
+    //TODO calibrate
+    double actualCurrent = holderMotor.getOutputCurrent();
+    double power = 0.4;
+    double expectedCurrent = 9.375 * power - 0.6384;
```

Comment:
double power = 0.4; 
why the power is 0.4? make another class variable and save in it the current power that given to the motors.